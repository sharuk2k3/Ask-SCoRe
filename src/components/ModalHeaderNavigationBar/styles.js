import {StyleSheet, Dimensions} from "react-native";

let deviceWidth = Dimensions.get("window").width;
let deviceHeight = Dimensions.get("window").height;

export default StyleSheet.create({
  navigationBar: {
    backgroundColor: "#51AD28",
    height: 40,
    flexDirection: "row",
    justifyContent: "space-between",
    alignItems: "center",
    width: "100%",
    top: 0,
  },

  leftIconContainer: {
    marginLeft: 10,
    marginTop: 5,
    borderRadius: 5,
  },

  rightIconContainer: {
    marginLeft: 10,
    marginTop: 5,
    borderRadius: 5,
  },

  titleArea: {
    justifyContent: "center",
    alignItems: "center",
  },

  goBackArea: {
    justifyContent: "center",
    alignItems: "center",
    marginLeft: 5,
  },

  titleFont: {
    fontSize: 18,
    color: "white",
  },
});
