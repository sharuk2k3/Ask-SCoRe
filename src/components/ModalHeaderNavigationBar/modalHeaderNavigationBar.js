import React, {Component} from "react";
import {Text, View, TouchableOpacity} from "react-native";
import styles from "./styles";
import Ionicons from "react-native-vector-icons/FontAwesome";

export default class modalHeaderNavigationBar extends Component {
  render() {
    return (
      <View style={styles.navigationBar}>
        <View style={styles.goBackArea}>
          <TouchableOpacity onPress={this.props.onPress}>
            <Ionicons name={"arrow-circle-left"} size={25} color={"white"} />
          </TouchableOpacity>
        </View>
        <View style={styles.titleArea}>
          <Text style={styles.titleFont}>{this.props.title}</Text>
        </View>
        <View style={styles.goBackArea}>
          <Text style={styles.titleFont}> </Text>
        </View>
      </View>
    );
  }
}
