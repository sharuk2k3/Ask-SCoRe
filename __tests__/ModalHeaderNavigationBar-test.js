import "react-native";
import React from "react";
import ModalHeaderNavigationBar from "../src/components/ModalHeaderNavigationBar/modalHeaderNavigationBar";
import renderer from "react-test-renderer";

jest.useFakeTimers();

test("renders Question Screen correctly", () => {
  const tree = renderer.create(<ModalHeaderNavigationBar />).toJSON();
  expect(tree).toMatchSnapshot();
});
