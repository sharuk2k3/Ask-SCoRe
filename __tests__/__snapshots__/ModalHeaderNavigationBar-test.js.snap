// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Question Screen correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#51AD28",
      "flexDirection": "row",
      "height": 40,
      "justifyContent": "space-between",
      "top": 0,
      "width": "100%",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "justifyContent": "center",
        "marginLeft": 5,
      }
    }
  >
    <View
      accessible={true}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <Text
        allowFontScaling={false}
        style={
          Array [
            Object {
              "color": "white",
              "fontSize": 25,
            },
            undefined,
            Object {
              "fontFamily": "FontAwesome",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {},
          ]
        }
      >
        ï‚¨
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "justifyContent": "center",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "white",
          "fontSize": 18,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "justifyContent": "center",
        "marginLeft": 5,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "white",
          "fontSize": 18,
        }
      }
    >
       
    </Text>
  </View>
</View>
`;
